js/gl.js

```diff
@@ -424,10 +424,10 @@ const SAPP_MODIFIER_SUPER = 8;
 
 function into_sapp_mousebutton(btn) {
     switch (btn) {
-        case 0: return 0;
```

Comment:
Hmm, maybe its easier to fix this on the rust side? While technically all gl.js's should be versioned and updated, I am pretty sure lots of people just use the last one and this will break existing code quite a lot :(

Reply:
Not sure about easier since that will touch all other platforms (I think?), but I think you're right in that rust updates don't require also getting a new .js file every time, will get on it!